time time
uint8 seqNum
uint8 imuId
AccelGyroMag data
